app/renderer/actions/Session.js

```diff
@@ -208,9 +210,25 @@ export function newSession (caps, attachSessId = null) {
       case ServerTypes.headspin:
         host = session.server.headspin.hostname;
         port = session.server.headspin.port;
-        path = `/v0/${session.server.headspin.apiKey}/wd/hub`;
+        path = `/wd/hub`;
```

Comment:
Why are you changing things for Headspin's integration?

Reply:
my mistake, will fix it now